src/views/templates/view_steps_password.jade

```diff
@@ -1,5 +1,6 @@
 extends _base
 
+
```

Comment:
Same remark : this template should not be affected by this PR, especially for code cleaning/reformatting.